flows/yosys/synth.tcl

```diff
@@ -0,0 +1,30 @@
+# Default Yosys synthesis tcl script for the yosys_synth_file_step rule.
+# It can be replaced by a user-defined script by overriding the synth_tcl
+# argument of that rule.
+
+yosys -import
+
+# read design
+set input_rtl $::env(INPUT_RTL)
+yosys read_verilog -sv -defer $input_rtl
+
+# generic synthesis
+set top $::env(CONSTANT_TOP)
+yosys synth -top $top
+
+# mapping to liberty
+set liberty $::env(LIBERTY)
+dfflibmap -liberty $liberty
+
+#if { [info exists ::env(CLOCK_PERIOD) ] } {
```

Comment:
Maybe avoid commented code?

Reply:
Fixed (at least partially, as noted in the TODO it isn't tested completely)